packages/database/external.yml

```diff
@@ -303,3 +303,19 @@
   audience: students, pro devs
   level: intermediate
   tags: github copilot, node.js, azure, express, javascript, typescript, app service, cosmos db, github actions, github codespaces
+
+  - title: Unleash the Power of Azure Open AI
+  url: https://microsoft.github.io/moaw/workshop/gh:microsoft/azurechat/main/docs/
```

Comment:
link does not work: it's not a workshop in MOAW format, so it should be a link to a website